cypress/integration/basic_commands.spec.ts

```diff
@@ -49,4 +46,45 @@ describe('Basic Commands Test', () => {
       cy.get('header .username').should('have.text', username),
     );
   });
+
+  it('Should create a problem', () => {
+    const username = uuid();
+    const password = uuid();
+    const problemAlias = uuid().slice(0, 10); // Too large for the alias
+    const tag = 'Recursion';
+    const autoCompleteTextTag = 'recur';
+    const problemLevelIndex = 0;
+
+    cy.register({ username, password });
+    // Select problem nav
+    cy.get('[data-nav-problems]').click();
+    cy.get('[data-nav-problems-create]').click();
+    // Fill basic problem form
+    cy.get('[name="title"]').type(problemAlias).blur();
+
+    // Alias should be the same as title.
+    cy.get('[name="problem_alias"]').should('have.value', problemAlias);
+
+    cy.get('[name="source"]').type(problemAlias);
+    cy.get('[name="problem_contents"]').attachFile('testproblem.zip');
+    cy.get('[data-tags-input]').type(autoCompleteTextTag);
+
+    // Tags panel
+    cy.waitUntil(() =>
+      cy
+        .get('[data-tags-input] a:first')
```

Comment:
```suggestion
        .get('[data-tags-input] .vbt-autcomplete-list a.vbst-item:first')
```

en este caso en particular `a:first` funciona porque lo que queremos es el primer elemento.